### PR TITLE
casper 1.4.5 protocol

### PR DESCRIPTION
**Pull request by sacherjj**
activation point 4417
2022-04-04 13:45:48  UTC
2022-04-04 06:45:48  US/Pacific
2022-04-04 09:45:48  US/Eastern
2022-04-04 15:45:48  EU/Zurich
2022-04-04 21:45:48  ASIA/HK

cleaned up known_addresses to remove missing/lagging nodes.
Updated max_global size to 2 TB

### CODE REVIEW

File: stage_upgrade.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script will stage the upgrade files for casper network from 1.4.3 to 1.4.4
```
Comment (EdHastingsCasperAssociation) [inline stage_upgrade.sh]: this says 1.4.3 to 1.4.4 but should it be 1.4.4 to 1.4.5 ?

**Review by EdHastingsCasperAssociation — APPROVED** [review]

File: stage_upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  exit 1
+fi
+
+if [[ -d "/etc/casper/1_4_4" ]]; then
+   echo "Upgrade 1.4.4 already staged."
+   exit 0
+fi
+
+# bump max global state for protocol 1_4_4:
+sudo sed -i "/max_global_state_size =/c\max_global_state_size = 1_539_316_278_886" /etc/casper/1_4_4/config.toml
```
Comment (EdHastingsCasperAssociation) [inline stage_upgrade.sh]: Is 1.4.4 correct here?

File: stage_upgrade.sh (hunk 1 above)
Comment (sacherjj) [inline stage_upgrade.sh]:
```suggestion
# This script will stage the upgrade files for casper network from 1.4.4 to 1.4.5
```

Comment (sacherjj) [reply stage_upgrade.sh]: Yep, corrected.

File: stage_upgrade.sh (hunk 2 above)
Comment (sacherjj) [reply stage_upgrade.sh]:
```suggestion
if [[ -d "/etc/casper/1_4_5" ]]; then
   echo "Upgrade 1.4.5 already staged."
   exit 0
fi

# bump max global state for protocol 1_4_4:
sudo sed -i "/max_global_state_size =/c\max_global_state_size = 1_539_316_278_886" /etc/casper/1_4_4/config.toml
```

**Review by thamps-casp — APPROVED** [review]

**Pull request merged**